internal/generate/mysql/model.go.tmpl

```diff
@@ -137,7 +137,7 @@ WHERE TRUE
 var {{ .CamelName }}DeleteSql = `
 DELETE FROM {{ .Table.SchemaName }}.{{ .Table.TableName }}
 WHERE TRUE
-{{- range $selectFields }}
+{{- range $pkFields }}
```

Comment:
like i commented earlier on the chain, we're assuming there is always a primary key. I personally have a production use case of multiple id columns co-existing (business use case) and need a way to lookup a single row with this alternate id!

with the existing approach, both scenarios can be accommodated while this accommodates only one!